dev_guide/projects.adoc

```diff
@@ -165,3 +165,29 @@ To delete a project using the CLI:
 ----
 $ oc delete project <project_name>
 ----
+
+[[assigning-pods-to-specific-nodes]]
+
+== Assigning Pods to Specific Nodes
+
+You can use node selectors in conjunction with labeled nodes to control pod placement.
+
+[NOTE]
+====
+OpenShift administrators can assign labels link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+====
+
+OpenShift cluster administrators can set the cluster-wide default node selector to restrict pod placement to specific nodes. As an OpenShift developer, you can set a node selector on a project configuration to restrict nodes even further. This will be added to the cluster-wide node selector, meaning it will be more restrictive.
+
+To add a node selector, edit a pod configuration, and add the `nodeSelector` value:
+
+====
+----
+...
+nodeSelector:
+  type:user-node,region:east
```

Comment:
We're trying not to add links to upstream too much these days. I think the example works. And there was info about the admin-set ones in there already, but I changed it around a little and it should be better now.